package.json

```diff
@@ -33,6 +33,7 @@
 		"node": ">=8.6"
 	},
 	"scripts": {
+		"start" : "mkdocs serve",
```

Comment:
is this really part of this PR or should it be moved to another PR?

Reply:
NTC - already described as part of the PR - It is useful and don't see the need to raise a separate issue for a one-liner like this.

Reply:
we can leave as it is, but in the future one issue, one PR, please, even if they are one liners :)

thanks!